i18n/en.json

```diff
@@ -2375,8 +2375,8 @@
   "get_app.dontHaveTheDesktopApp": "Don't have the Desktop App?",
   "get_app.dontHaveTheMobileApp": "Don't have the Mobile App?",
   "get_app.downloadTheAppNow": "Download the app now.",
-  "get_app.ifNothingPrompts": "You can view it in the{siteName} desktop app or continue in the web browser.",
-  "get_app.ifNothingPromptsMobile": "You can view it in the{siteName} mobile app or continue in the web browser.",
+  "get_app.ifNothingPrompts": "You can view {siteName} desktop app or continue in the web browser.",
+  "get_app.ifNothingPromptsMobile": "You can view {siteName} mobile app or continue in the web browser.",
```

Comment:
@amyblais I'm not sure this wording makes sense?

Reply:
It was suggested by community, I'll close this PR and assign the ticket for you.